### PR TITLE
silence warnings

### PR DESCRIPTION
**Pull request by grosser**
@apanzerj @irwaters @jonmoter

### CODE REVIEW

**Comment by jonmoter** [comment]
Which warnings are these changes silencing?

**Comment by grosser** [comment]
`ruby -v` warnings ... use of uninitialized instance vars and bad regexes

File: app/models/build.rb [hunk 1]
```diff
@@ -2,7 +2,7 @@
 class Build < ActiveRecord::Base
   SHA1_REGEX = /\A[0-9a-f]{40}\Z/i
   SHA256_REGEX = /\A(sha256:)?[0-9a-f]{64}\Z/i
-  DIGEST_REGEX = /\A[\w._-]+\/[\w\/_-]+@sha256:[0-9a-f]{64}\Z/i
+  DIGEST_REGEX = /\A[\w\.-]+\/[\w\/-]+@sha256:[0-9a-f]{64}\Z/i
```
Comment (sandlerr) [inline app/models/build.rb]: dots are already literal in this context

Comment (grosser) [inline app/models/build.rb]: ah yeah ... must have been the _ that triggered the warning I saw ... `.` does not cause a warning ... removed ... even though I think `\.` might be less ambiguous

**Comment by grosser** [comment]
I can haz +1 ?

**Comment by sandlerr** [comment]
so this just fixes variable shadowing and `_` already being part of `\w`? :+1: to those changes

**Comment by grosser** [comment]
yep!

**Pull request merged**